standalone/src/main/java/io/pravega/local/InProcPravegaCluster.java

```diff
@@ -101,15 +101,19 @@
 
     private String zkUrl;
     private boolean startRestServer = true;
+    private String userName;
+    private String passwd;
+    private String certFile;
+    private String keyFile;
+    private String passwdFile;
 
     @Builder
     public InProcPravegaCluster(boolean isInProcZK, String zkUrl, int zkPort, boolean isInMemStorage,
```

Comment:
This constructor's arg list is getting pretty big. Should we consider just passing in `SingleNodeConfig` and picking the values from there? That will also help reduce the number of private fields since we can just hold a pointer to the config object.

Reply:
Using class level lombok builder pattern to ease this.